castle/cms/browser/controlpanel/templates/status.pt

```diff
@@ -0,0 +1,49 @@
+
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en"
+      xmlns:tal="http://xml.zope.org/namespaces/tal"
+      xmlns:metal="http://xml.zope.org/namespaces/metal"
+      xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+      lang="en"
+      metal:use-macro="context/prefs_main_template/macros/master"
+      i18n:domain="plone">
+<body>
+<metal:main metal:fill-slot="prefs_configlet_main">
+
+    <a href=""
+       id="setup-link"
+       tal:attributes="href string:$portal_url/plone_control_panel"
+       i18n:translate="">
+        Site Setup
+    </a>
+
+
+
+    <h1> Subprocesses </h1>
+
+    <div metal:use-macro="context/global_statusmessage/macros/portal_message">
+      Portal status message
+    </div>
+
+    <div id="content-core">
+        <div tal:condition="python:view.docsplit()[0]">
```

Comment:
Rather than evaluate this Python expression twice, you could wrap both divs in one more div in which you use tal:define to assign the value of the Python expression to a variable, and then in each of these contained divs you use the variable. Same applies below.

Reply:
Take a look at the site crawler control panel page template, https://github.com/castlecms/castle.cms/blob/59c6c52d6f5d8bf21cca92caf0e432f4312b4ab8/castle/cms/browser/controlpanel/templates/crawler.pt for ideas on how to make this one look more like the others